src/MusicStore.Standalone/Program.cs

```diff
@@ -44,9 +44,10 @@ public static void Main(string[] args)
                 builder.UseKestrel();
             }
 
-            var host = builder.Build();
-
-            host.Run();
+            using (var host = builder.Build())
```

Comment:
Ironically, I am in the process of creating a pull request to address those specifically. Those templates should all be utilizing the `using` statement.